repositories.tf

```
@@ -270,3 +270,29 @@ module "repo_osac_test_infra" {
     }
   ]
 }
+
+module "repo_osac_templates" {
+  source      = "./modules/common_repository"
+  visibility  = "public"
+  name        = "osac-templates"
+  description = "Ansible roles for base/generic templates."
```

I would probably replace this with:


```suggestion
  description = "OSAC base templates"
```